packages/ui/LoadingWrapper/index.tsx

```diff
@@ -4,7 +4,7 @@ import { ErrorComponent } from "../ErrorComponent";
 import LoadingSpinner from "../LoadingSpinner";
 
 type AccessWrapperProps = {
-  isLoading: boolean;
+  isLoading?: boolean;
```

Comment:
feel wrong to make `isLoading` optional for LoadingWrapper component. Are you sure thats the right approach?

Reply:
got rid of this component altogether since it was just adding an error component!